server/channels/app/web_broadcast_hooks.go

```diff
@@ -82,11 +82,8 @@ func usePostedAckHook(message *model.WebSocketEvent, postedUserId string, channe
 }
 
 func (h *postedAckBroadcastHook) Process(msg *platform.HookedWebSocketEvent, webConn *platform.WebConn, args map[string]any) error {
-	// Add if we have mentions or followers
-	// This works since we currently do have an order for broadcast hooks, but this probably should be reworked going forward
-	if msg.Get("followers") != nil || msg.Get("mentions") != nil {
-		msg.Add("should_ack", true)
-		incrementWebsocketCounter(webConn)
+	// Don't ACK mobile app websocket events at this time, we may revisit this when we can add this to mobile app
+	if webConn.GetSession().IsMobile() {
```

Comment:
Not sure that we can guarantee `webConn.GetSession()` won't ever return `nil`. May be worth checking for it, and log a warning if that happens.

Reply:
I don't know if it's actually in the code, but I know there was some discussion at some point about WebSocket connections for non-authenticated users. I don't know if those users would have a `WebConn` or if that was even part of MM itself though since some of that was Focalboard-related